app/src/androidTest/java/org/sagebionetworks/research/mpower/MedicationTimeBasedFilteringTest.kt

```
@@ -72,8 +73,10 @@ class MedicationTimeBasedFilteringTest {
         val CONFIG_3: MedicationLog
 
         init {
-            val timeStamp1 = MedicationTimestamp.builder().setTimeOfDay("13:00")!!.build()
-            val timeStamp2 = MedicationTimestamp.builder().setTimeOfDay("18:00")!!.build()
+            val timeStamp1 = MedicationTimestamp.builder().setTimeOfDay("13:00")?.build()
```

I think for tests it would be better to leave the !! and fail early with a nullpointer.

---

fixed